scripts/update/jellyfin.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+#
+# source the functions we need for this script.
+. /etc/swizzin/sources/functions/utils
+# Get our main user credentials using a util function.
+username="$(_get_master_username)"
+#
+# remove the old service and remove legacy files.
+if [[ -f /install/.jellyin.lock && -f /etc/systemd/system/jellyfin.service ]]; then
+    systemctl -q disable --now jellyfin.service
+    rm_if_exists /etc/systemd/system/jellyfin.service
+    kill -9 $(ps xU ${username} | grep "/opt/jellyfin/jellyfin -d /home/${username}/.config/Jellyfin$" | awk '{print $1}') >/dev/null 2>&1
+    rm_if_exists /opt/jellyfin
+    rm_if_exists /opt/ffmpeg
+fi
+#
+# data migration to default locations.
+if [[ -f /install/.jellyin.lock && -d "/home/${username}/.config/Jellyfin/config" ]]; then
+    mkdir -p /etc/jellyfin
+    mkdir -p /var/lib/jellyfin/{data,root,metadata}
+    #
+    [[ -d "/home/${username}/.config/Jellyfin/config" ]] && cp -fRT "/home/${username}/.config/Jellyfin/config" /etc/jellyfin
+    rm_if_exists /etc/jellyfin/encoding.xml
+    [[ -d "/home/${username}/.config/Jellyfin/data" ]] && cp -fRT "/home/${username}/.config/Jellyfin/data" /var/lib/jellyfin/data
+    [[ -d "/home/${username}/.config/Jellyfin/metadata" ]] && cp -fRT "/home/${username}/.config/Jellyfin/metadata" /var/lib/jellyfin/metadata
+    [[ -d "/home/${username}/.config/Jellyfin/root" ]] && cp -fRT "/home/${username}/.config/Jellyfin/root" /var/lib/jellyfin/root
+    #
+    sed -r 's#<PublicPort>(.*)</PublicPort>#<PublicPort>8096</PublicPort>#g' -i /etc/jellyfin/system.xml
+    sed -r 's#<PublicHttpsPort>(.*)</PublicHttpsPort>#<PublicHttpsPort>8920</PublicHttpsPort>#g' -i /etc/jellyfin/system.xml
+    sed -r 's#<HttpServerPortNumber>(.*)</HttpServerPortNumber>#<HttpServerPortNumber>8096</HttpServerPortNumber>#g' -i /etc/jellyfin/system.xml
+    sed -r 's#<HttpsPortNumber>(.*)</HttpsPortNumber>#<HttpsPortNumber>8920</HttpsPortNumber>#g' -i /etc/jellyfin/system.xml
+    sed -r 's#<RequireHttps>false</RequireHttps>#<RequireHttps>true</RequireHttps>#g' -i /etc/jellyfin/system.xml
+    #
+    if [[ -f /etc/nginx/apps/jellyfin.conf ]]; then
+        sed -r 's#proxy_pass https://127.0.0.1:(.*);#proxy_pass https://127.0.0.1:8920;#g' -i /etc/nginx/apps/jellyfin.conf
+        systemctl -q restart nginx.service
+    fi
+    #
+    rm_if_exists "/home/${username}/.config/Jellyfin"
+    rm_if_exists "/home/${username}/.cache/jellyfin"
+    rm_if_exists "/home/${username}/.aspnet"
+fi
+#
+if ! check_installed jellyfin; then
```

Comment:
Without the lockfile condition at the top, this will just install jellyfin for everyone if they run `box update`

Reply:
/resolved